### DISCUSSION TITLE
CSV connection field lengths all 32767

### DISCUSSION THREAD

**Discussion by davidcwhitney** [dbeaver/dbeaver#9027] [Q&A] [answered]
Why does every field in my tab- and comma-delimited text files have length 32767?  I want max 255 and I don't want to have to modify hundreds of fields one at a time.

Thanks.
![image](https://user-images.githubusercontent.com/67126011/85075812-da4c2480-b173-11ea-81f6-a494c577b602.png)
System information:  
- Operating system Windows 10.0.17134
- DBeaver version Enterprise 6.3.0

Connection specification:
- CSV both comma and tab delimited

**Comment by kseniiaguzeeva** [comment, marked as answer]
This is not DBeaver issue, but CSV driver characteristic.
